src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -143,17 +145,55 @@
                                               :value (get (:attributes @state) property-kwd)
                                               :onChange update-property
                                               :rows 3})
+                     (= typeahead "ontology")
+                     (style/create-text-field {:ref property-kwd
+                                               :className "typeahead"
+                                               :placeholder "Select an ontology."
+                                               :style {:width "100%"}
+                                               :value (get (:attributes @state) property-kwd nil)
+                                               :onChange update-property
+                                               })
                      :else
                      (style/create-text-field {:style (colorize {:width "100%"})
                                                :type (cond (= renderHint "date") "date"
                                                            (= type "integer") "number"
                                                            :else "text")
                                                :min minimum
+                                               :disabled disabled?
                                                :placeholder inputHint
                                                :value (get (:attributes @state) property-kwd)
-                                               :onChange update-property}))]))
-          questions)]))})
 
+                                               :onChange update-property}))]))
+          questions)]))
+   :component-did-mount
+   (fn [{:keys [props state refs]}]
+     (let [options (js/Bloodhound.
+                     (clj->js
+                       {:datumTokenizer js/Bloodhound.tokenizers.whitespace
+                        :queryTokenizer js/Bloodhound.tokenizers.whitespace
+                        :remote (clj->js {:url (str (config/api-url-root) "/duos/autocomplete/%QUERY")
+                                          :wildcard "%QUERY"})}))
+           existing-ontology (get-in props [:workspace :workspace :library-attributes :library:diseaseOntologyLabel])
+           property-kwd (@refs "library:diseaseOntologyLabel")] ;; fix this
```

Comment:
`library:diseaseOntologyLabel` in the previous two lines should not be hardcoded; it needs to be passed in as an argument for any given property.